style.css

```diff
@@ -0,0 +1,1238 @@
+    color: white;
+}
+
+.dropdown__link:hover {
+    color: #55BBA6;
+}
+
+.dropdown__item::after {
+    content: " ";
+    display: flex;
+    width: 20px;
+    height: 1px;
+    background: white;
+    opacity: 0.15;
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+.dropdown__item:last-child::after {
+    content: " ";
+    background-color: #272626;
+}
+
+.nav__link--dropdown {
+    font-family: "CircularStd-Book";
+    font-size: 18px;
+    letter-spacing: 0.92px;
+    line-height: 1.3;
+    color: white;
+    text-decoration: none;
+}
+
+.nav__link--dropdown:after {
+    content: " ";
+    border: solid white;
+    border-width: 0 2px 2px 0;
+    display: inline-block;
+    padding: 3px;
+    margin-left: 17px;
+    margin-right: 17px;
+    transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+}
+
+.nav__link--dropdown:hover {
+    cursor: pointer;
+}
+
+.hero {
+    background-image: radial-gradient(58.3% 58.3% at 75.33% 44.51%, rgba(0, 0, 0, 0.0001) 0%, rgba(5, 4, 2, 0.6) 100%), url(hero-image.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    top: -21px;
+    position: relative;
+    padding-bottom: 16.5rem;
+    left: 90px;
+    margin: 0px;
+    -webkit-transition: background-image 0.2s ease-in-out;
+    transition: opacity 1s ease-in-out;
+}
+
+.hero__heading {
+    font-family: "CircularStd-Black";
+    font-size: 0.88rem;
+    letter-spacing: 2.42px;
+    color: white;
+    text-transform: uppercase;
+    max-width: 335px;
+    max-height: 20px;
+    margin-bottom: 0px;
+}
+
+.hero__heading::before {
+    content: " ";
+    width: 27px;
+    height: 4px;
+    background-color: #55BBA6;
+    display: block;
+    margin-bottom: 15px;
+}
+
+.hero__title {
+    font-family: "DMSerifDisplay-Regular";
+    color: white;
+    font-size: 7.2rem;
+    letter-spacing: 19.64px;
+    text-transform: uppercase;
+    max-width: 584px;
+    max-height: 178px;
+    margin: 0px;
+    line-height: 1.4;
+}
+
+
+.hero__content {
+    font-family: "SofiaPro-Medium";
+    color: white;
+    font-size: 1rem;
+    letter-spacing: 1.72px;
+    padding-right: 180px;
+}
+
+.hero__undernote {
+    font-family: "SofiaPro-Regular";
+    color: white;
+    font-size: 0.66rem;
+    letter-spacing: 0.7px;
+    max-width: 447px;
+    max-height: 31px;
+    margin-top: 29px;
+    margin-bottom: 70px;
+    line-height: 2.41;
+}
+
+.hero__cta {
+    display: flex;
+}
+
+.hero__button {
+    background: #55BBA6;
+    font-family: "SofiaPro-Medium";
+    font-size: 18px;
+    letter-spacing: 0.7px;
+    line-height: 1.33;
+    color: white;
+    border: 2px solid #55BBA6;
+    padding: 22px 64px 14px 63px;
+    max-width: 250px;
+    max-height: 60px;
+    text-decoration: none;
+    display: block;
+    margin-right: 21px;
+    border-radius: 2px;
+}
+
+.hero__button:hover {
+    background: #399280;
+}
+
+.hero__button--transparent {
+    background: rgba(255, 255, 255, 0.2);

```

Comment:
ar fi mai ok sa il transformi in hex aici, sunt site-uri unde poti cauta transformarea